### DISCUSSION TITLE
パスワードセーブを「セーブデータをまるごとエクスポートする機能」として再定義したい

### DISCUSSION THREAD

**Discussion by matsuyuki-a** [WWAWing/WWAWing#555] [Ideas]
パスワードセーブは、現状1つのゲームデータの状態を保存する機能であるが、ブラウザ(IndexedDB)への保存機能の実装により、使い所があまりなくなってきている

一方で、ブラウザに保存された4つのセーブデータを他のブラウザやPCで遊びたいときにエクスポートする機能がないという問題がある。

パスワードセーブは、「4つのQuickSave, オートセーブ」を含めたWWA全体の状態をエクスポートする機能として再定義できないか検討したい

**Comment by enjoyablespage** [comment]
Twitter連携などで出来ないですかね？

**Comment by matsuyuki-a** [comment]
どちらかというと、ここではローカルへのパスワードの保存について（not サーバへの保存) 改善できないかなと思っています。サーバへのセーブとかをTwitter連携でやるとかも確かに面白そうですが、別途の対応としたいです 🙇
